File: src/main/java/dev/sky_lock/pocketlifevehicle/util/Formats.java
package dev.sky_lock.pocketlifevehicle.util;

import org.bukkit.ChatColor;

/**
 * @author sky_lock
 */

public class Formats {

    public static String truncateToOneDecimalPlace(float f) {
        return String.format("%.1f", f);
    }

    public static String removeBlanks(String blankText) {
        return blankText.replaceAll("\\s", "");
    }

    public static String colorize(String text) {
        return ChatColor.translateAlternateColorCodes('&', text);
    }

}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/util/Formats.kt
package dev.sky_lock.pocketlifevehicle.util

import org.bukkit.ChatColor

/**
 * @author sky_lock
 */
object Formats {
    fun truncateToOneDecimalPlace(f: Float): String {
        return String.format("%.1f", f)
    }

    fun removeBlanks(blankText: String): String {
        return blankText.replace("\\s".toRegex(), "")
    }

    fun colorize(text: String): String {
        return ChatColor.translateAlternateColorCodes('&', text)
    }
}